manager/integration/tests/test_ha.py

```diff
@@ -2231,6 +2231,7 @@ def test_auto_remount_with_subpath(client, core_api, storage_class, sts_name, st
                             namespace='longhorn-system',
                             wait=True)
         wait_for_volume_healthy(client, vol_name)
+        common.wait_for_pod_phase(core_api, pod_name, pod_phase="Pending")
         common.wait_for_pod_remount(core_api, pod_name, chk_path=data_path)
```

Comment:
`wait_for_pod_remount` has retry right? why it can not succeed after retries?

The same situation also happens to step 9 but you didn't handle it.

Reply:
Sounds good.